round4/team-01.md

```diff
@@ -0,0 +1,118 @@
+# Chrome Extension Idea: PennCourse Extension
+
+## Authors
+
+Eshaan Chichula, Franci Branda-Chen, Jake Garrison Murphy, Matt fu
+
+## Problem Statement
+
+Our Chrome Extension simplifies the course selection process for Penn students by addressing a specific pain point: the need to switch back and forth between Path@Penn and Penn Course Review to assess course difficulty and instructor quality. By integrating this critical information directly into the Path@Penn interface, our tool minimizes the friction involved in course research, allowing students to make informed decisions with ease.
+
+## Target Audience
```

Comment:
I really like how you've narrowed down on the target audience instead of creating a very broad/generic app!